lib/local-links-manager/import/services_tier_importer.rb

```diff
@@ -72,11 +72,11 @@ def checked_services
       def check_for_missing_services(summariser)
         missing = []
         Service.enabled.each do |service|
-          if checked_services.exclude?(service)
-            summariser.increment_missing_record_count
-            missing << service
-            ServiceTier.where(service: service).destroy_all
-          end
+          next unless checked_services.exclude?(service)
```

Comment:
```suggestion
          next if checked_services.include?(service)
```

These next cops create some confusing double negatives sometimes.

Reply:
Updated 👍